### PR TITLE
update maintenance and trouble page static content

### PR DESCRIPTION
**Pull request by benjiwheeler**
### Resolves:

Resolves https://github.com/LLK/scratch-www/issues/4914

### Changes:

* take "fixing a problem" content we previously had in maintenance.html, and copy it to trouble.html
* update language in each to be clearer about their distinct meaning, per https://docs.google.com/document/d/1D8gBGfrtEKTsiOpFiXbNzQNjZF4WHOld0nchtgZIHZA/edit
* update design with new images in S3 bucket, per Figma designs https://www.figma.com/file/L82ypZomFB9t6kcjRtesTV/Trouble-%2F-Maintenance-Page?node-id=203%3A45

Note that this PR will not make these pages actually be used! These files are just a convenient place to keep the static html/css/js content that we will paste into our Fastly services' synthetic responses.

### Trouble page

image

image

image

### Maintenance page

image

image

image

### CODE REVIEW

**Comment by Accio1** [comment]
This is neat! I really like the little flowerpot and rock icons!

**Review by Naleksuh — CHANGES_REQUESTED** [review]
This URL is outdated and should be changed to https://downloads.scratch.mit.edu/desktop/Scratch%20Setup.exe

**Comment by Naleksuh** [comment]
> 
> 
> This URL is outdated and should be changed to https://downloads.scratch.mit.edu/desktop/Scratch%20Setup.exe

For some reason this didn't show what URL I was talking about. For clarity I meant the URL on line 203 of static/cdn/trouble.html

**Comment by benjiwheeler** [comment]
blocked on getting design ok; then i'll need an engineer to review

**Comment by benjiwheeler** [comment]
Design gave ok, thanks @kathymakes!

File: static/cdn/maintenance.html [hunk 1]
```diff
@@ -239,12 +242,12 @@
       </div>
       <div class="main">
         <div class="content">
-          <h1>The Scratch Team is working hard to fix an issue with the Scratch website</h1>
-          <h2 class="reload" onclick="location.reload()">Try reloading the page in a few minutes</h2>
+          <img src="http://scratch-maintenance.s3.amazonaws.com/cairn.svg" />
```
Comment (benjiwheeler) [inline static/cdn/maintenance.html:245]: added this image to s3 bucket

File: static/cdn/trouble.html [hunk 2]
```diff
@@ -1,208 +1,332 @@
...
       </div>
-      <div class="box-content">
-        <div class="twitter-containter">
-          <img src="http://scratch-maintenance.s3.amazonaws.com/site-down.png" style="margin-left:100px;padding:10px;float:left;"/>
-          <div class="twitter-feed">
-            <a class="twitter-timeline" href="https://twitter.com/scratch" height="200" width="400" data-widget-id="534361794572058626" data-chrome="noheader nofooter transparent noborders" data-tweet-limit="1">Tweets by @scratch</a>
-            <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+      <div class="main">
+        <div class="content">
+          <img src="http://scratch-maintenance.s3.amazonaws.com/wateringcan.svg" />
```
Comment (benjiwheeler) [inline static/cdn/trouble.html:245]: added this image to s3 bucket

**Review by kchadha — APPROVED** [review]
LGTM

**Review by Naleksuh — APPROVED** [review]
Link seems updated now

**Comment by benjiwheeler** [comment]
FYI: I updated the maintenance and trouble page response content in both the scratch-www-staging and scratch-www-production fastly services.

**Pull request merged**